ml-proto/spec/eval.ml

```diff
@@ -109,7 +109,7 @@ let mem_size v at =
  * we currently only support i32, just test that.
  *)
 let mem_overflow x =
-  I64.gt_u x (Int64.of_int32 Int32.max_int)
+  I64.gt_u x (I64_convert.extend_u_i32 Int32.minus_one)
```

Comment:
When I make that change, I get:

File "spec/eval.ml", line 117, characters 14-38:
Error: This expression has type I32.t -> I64.t
       but an expression was expected of type int

which I don't understand.